units/systemd-udevd@.service.in

```diff
@@ -0,0 +1,42 @@
+#  SPDX-License-Identifier: LGPL-2.1+
```

Comment:
maybe rename the file to contain the idea that it's network-namespace related in the name of the unit ?

Reply:
I noticed this unit does not work... Dropped.